From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Dreeam <61569423+Dreeam-qwq@users.noreply.github.com>
Date: Tue, 21 May 2024 11:43:49 +0800
Subject: [PATCH] Rebrand

This patch is referring to the following patch:
"Gale branding changes"
By: Martijn Muijsers <martijnmuijsers@live.nl>
As part of: GaleMC (https://github.com/GaleMC/Gale)
Licensed under: GPL-3.0-only (https://www.gnu.org/licenses/gpl-3.0.html)

diff --git a/src/main/java/io/papermc/paper/ServerBuildInfo.java b/src/main/java/io/papermc/paper/ServerBuildInfo.java
--- a/src/main/java/io/papermc/paper/ServerBuildInfo.java
+++ b/src/main/java/io/papermc/paper/ServerBuildInfo.java
@@ -24,6 +24,18 @@ public interface ServerBuildInfo {
     Key BRAND_GALE_ID = Key.key("galemc", "gale");
     // Gale end - branding changes

+    // Leaf start - Rebrand
+    /**
+     * The brand id for Pufferfish.
+     */
+    Key BRAND_PUFFERFISH_ID = Key.key("pufferfish", "pufferfish");
+    /**
+     * The brand id for Leaf.
+     */
+    Key BRAND_LEAF_ID = Key.key("winds-studio", "leaf");
+    // Leaf end - Rebrand
+
     /**
      * Gets the {@code ServerBuildInfo}.
      *
diff --git a/src/main/java/org/bukkit/command/defaults/VersionCommand.java b/src/main/java/org/bukkit/command/defaults/VersionCommand.java
--- a/src/main/java/org/bukkit/command/defaults/VersionCommand.java
+++ b/src/main/java/org/bukkit/command/defaults/VersionCommand.java
@@ -215,7 +215,7 @@ public class VersionCommand extends BukkitCommand {
         String version = Bukkit.getVersion();
         // Paper start
         if (version.startsWith("null")) { // running from ide?
-            setVersionMessage(Component.text("Unknown version, custom build?", NamedTextColor.YELLOW));
+            setVersionMessage(Component.text("* Unknown version, custom build?", NamedTextColor.RED)); // Leaf - Rebrand - Purpur - Clean up version command output
             return;
         }
         setVersionMessage(getVersionFetcher().getVersionMessage(version));
